content/md/en/docs/tutorials/smart-contracts/prepare-your-first-contract.md

```diff
@@ -406,7 +404,7 @@ Since we are only reading from the blockchain state (we're not writing any new d
 avoid submitting an extrinsic.
 
 ```bash
-cargo contract call --contract 5GRAVvuSXx8pCpRUDHzK6S1r2FjadahRQ6NEgAVooQ2bB8r5 --message get --suri //Alice --dry-run
+cargo contract call --contract PUT_CONTRACT_ADDRESS_HERE --message get --suri //Alice --dry-run
```

Comment:
Can you make a note reminding readers that this values comes from the `cargo contract instantiate` step from earlier? 

```suggestion
cargo contract call --contract $INSTANTIATED_CONTRACT_ADDRESS --message get --suri //Alice --dry-run
```